packages/babel-generator/test/fixtures/parentheses/sequence-expressions/output.js

```diff
@@ -6,4 +6,6 @@ throw a, b, c;
 switch (a, b, c) {}
 for (a in b, c);
 while (a, b, c);
-!function () {}(), a();
\ No newline at end of file
+!function () {}(), a();
+a, b, c;
+a, b, c;
```

Comment:
It seems like the old behavior is better here.
Although we don't guarantee formatting by default, it would be nice if we could preserve the formatting for this case.

Reply:
In most cases where parentheses don't matter we drop them, for example https://babeljs.io/repl#?browsers=defaults%2C%20not%20ie%2011%2C%20not%20ie_mob%2011&build=&builtIns=false&corejs=3.21&spec=false&loose=false&code_lz=BQQwBAPhYEYJSWgYwNwChxTMGixLiA&debug=false&forceAllTransforms=false&modules=false&shippedProposals=false&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=env%2Creact%2Cstage-2&prettier=false&targets=&version=7.25.3&externalPlugins=&assumptions=%7B%7D

In general, people that care about parentheses should enable the `createParenthesizedExpression` option.

Reply:
I'm still a little concerned that this removes one node from the AST tree, but now I think this is OK.